### ISSUE TITLE
Multiples not working properly

### ISSUE THREAD

**Issue by BrandoCommando** [MosaicManufacturing/canvas-issues#17]
**Describe the bug**
It is possible to add multiples of the same model, but only after a different model has been added.

**To Reproduce**
1. Go to the "Place" tab.
2. Under "Object Browser", click "+" to add a model.
3. Trying to repeat the above steps (with the same model) results in failure.
4. If you add a different model, then go back and add the first model, it works. You just can't add the same model twice in a row.

**Expected behavior**
Printing multiples of the same model is critical for many printing applications. Canvas should either have the ability to duplicate a specific model, or at least be able to add it to the workspace multiple times.

**Operating System (please complete the following information):**
 - OS: Mac OS 10.13.6
 - Browser: Chrome
 - Version: 68

**Comment by brandonbloch** [comment]
Hi @BrandoCommando,

Thanks for reporting this! We actually recently identified and fixed the problem with uploading the same file(s) multiple times in a row. We will be rolling out an update that includes this fix by the end of the week.

As for duplicating one or more models, this is a planned feature but will take a little more time. If you'd like to keep track of the progress on that feature, please open a separate issue for it.

Cheers,
Brandon
